### PR TITLE
[google][mobileads] Updated to version 7.8.1

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs [hunk 1]
```diff
@@ -427,7 +445,7 @@ interface RewardBasedVideoAdDelegate
 	{
 		// @optional -(void)rewardBasedVideoAdDidReceiveAd:(GADRewardBasedVideoAd *)rewardBasedVideoAd;
 		[EventArgs ("RewardBasedVideoAd")]
-		[EventName ("ReceivedAd")]
+		[EventName ("AdReceived")]
```
Comment (bholmes) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Do we have to break API here on these EventNames?

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs [hunk 2]
```diff
@@ -438,7 +456,7 @@ interface RewardBasedVideoAdDelegate
 		void DidOpen (RewardBasedVideoAd rewardBasedVideoAd);
 
 		[EventArgs ("RewardBasedVideoAd")]
-		[EventName ("StartedPlaying")]
+		[EventName ("PlayingStarted")]
```
Comment (bholmes) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Do we have to break API here on these EventNames

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs [hunk 3]
```diff
@@ -456,7 +474,7 @@ interface RewardBasedVideoAdDelegate
 
 		// @optional -(void)rewardBasedVideoAd:(GADRewardBasedVideoAd *)rewardBasedVideoAd didRewardUserWithReward:(GADAdReward *)reward;
 		[EventArgs ("RewardBasedVideoAdReward")]
-		[EventName ("RewardedUser")]
+		[EventName ("UserRewarded")]
```
Comment (bholmes) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Do we have to break API here on these EventNames

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs (hunk 1 above)
Comment (SotoiGhost) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Not really, we can avoid this change.

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs (hunk 2 above)
Comment (SotoiGhost) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Not really, we can avoid this change.

File: Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs (hunk 3 above)
Comment (SotoiGhost) [inline Google.MobileAds/source/Google.MobileAds/ApiDefinition.cs]: Not really, we can avoid this change.

**Pull request merged**